### ISSUE TITLE
add option to return output properties in camel case

### ISSUE THREAD

**Issue by 75lb** [75lb/command-line-args#57]
Add option to return camel-case property names on the output object, e.g. 

```js
const commandLineArgs = require('./')
const optionDefinitions = [
  { name: 'option-one' }
]

const options = commandLineArgs(optionDefinitions, { camelCase: true })
console.log(options)

/*
returns:

{  optionOne: 'something' }

instead of : 

{  'option-one': 'something' }
*/
```

**Comment by stevenvachon** [comment]
What if we don't simply want a camel case change, but want a full rename? Example: `--prep` to `preparation`.

**Comment by 75lb** [comment]
@stevenvachon that can be achieved in a single line of app code - i'd rather not add and support too much sugar in the library. If it suddenly becomes a highly requested feature i'll re-assess. 👍

**Comment by 75lb** [comment]
`camelCase` option added and released in v5.0.0. See the [release notes](https://github.com/75lb/command-line-args/releases) and [the wiki for an example](https://github.com/75lb/command-line-args/wiki/camelCase).
